backstop_data/engine_scripts/puppet/page-objects/crm-page.js

```diff
@@ -104,6 +116,25 @@ module.exports = class CrmPage {
       this.engine.click(selector),
       this.engine.waitForNavigation()
     ]);
+    await this.waitForWYSIWYG();
+    await this.waitForDatePicker();
+    await this.fixLayoutForTables();
+  }
+
+  /**
+  * Waits for Modal form to compelete rendering after some link to open modal is clicked
+  *
+  * @param {String} selector - the css selector for the element to click and wait for modal.
+  */
+  async clickAndWaitForModal (selector) {
```

Comment:
except for `constructor`, which should always be the first method, please order the methods in alphabetical order

Reply:
Cool! Doing!